src/open_inwoner/openklant/cms_apps.py

```diff
@@ -0,0 +1,13 @@
+from django.utils.translation import gettext_lazy as _
+
+from cms.app_base import CMSApp
+from cms.apphook_pool import apphook_pool
+
+
+@apphook_pool.register
+class OpenklantApphook(CMSApp):
```

Comment:
I guess this matches the module name, but perhaps something referring to `ContactFormulier` would be a clearer name?


```suggestion
class ContactFormulierApphook(CMSApp):
```

Reply:
I get your point, since we're only creating this for the contact form. However, the name doesn't just reflect the module name, but, by convention, the name of the registered Django app for which the hook is created ("openklant" in this case).

Reply:
In that case we go for consistency. Resolving.